### PR TITLE
Completed html-web-page challenge.

### PR DESCRIPTION
**Pull request by Yang-Darren**
image

### CODE REVIEW

File: html-web-page/index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>&#36;30</td>
+        <td>&#36;50</td>
+        <td>&#36;60</td>
+      </tr>
+    </tbody>
+
+  </table>
+  <br />
```
Comment (BrettAlbright) [inline html-web-page/index.html]: Make sure to remove this `br` tag. They can only be used for text formatting.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed. Looks great!

File: html-web-page/index.html [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      </tbody>
+
+    </table>
+  </div>
+
+  <form>
+    <div>
+      <label for="subscribe">Like what you see? Become part of the JOTMC family today!</label>
+    </div>
+    <br />
```
Comment (BrettAlbright) [inline html-web-page/index.html:84]: Avoid these going forward!

**Review by Zhiwutian — APPROVED** [review]
Approved with informational comment

**Pull request merged**